src/material/icon/icon-registry.ts

```diff
@@ -132,7 +133,8 @@ export class MatIconRegistry implements OnDestroy {
   constructor(
     @Optional() private _httpClient: HttpClient,
     private _sanitizer: DomSanitizer,
-    @Optional() @Inject(DOCUMENT) document: any) {
+    @Optional() @Inject(DOCUMENT) document: any,
+    @Optional() private readonly _errorHandler?: ErrorHandler) {
```

Comment:
Can you add `@breaking-change 9.0.0 `_errorHandler` parameter to be made required` here and down below where we have the null check so that our tooling can tell us to clean it up eventually?